src/frontend/src/handler/create_sink.rs

```diff
@@ -718,7 +718,7 @@ fn derive_default_column_project_for_sink(
 
         // If users specified the columns to be inserted e.g. `CREATE SINK s INTO t(a, b)`, the expressions of `Project` will be generated accordingly.
         // The missing columns will be filled with default value (`null` if not explicitly defined).
-        // Otherwhise, e.g. `CREATE SINK s INTO t`, the columns will be matched by their order in `select` query and the target table.
+        // Otherwise, e.g. `CREATE SINK s INTO t`, the columns will be matched by their order in `select` query and the target table.
```

Comment:
🥵 sorry for that and it will be fixed in https://github.com/risingwavelabs/risingwave/pull/16663, you can apply the suggestion to prevent confliction
```suggestion
        // Otherwhise, e.g. `CREATE SINK s INTO t`, the columns will be matched by their order in `select` query and the target table.
```